docs/client_reference.rst

```diff
@@ -179,6 +179,11 @@ The client session supports the context manager protocol for self closing.
 
       Get proxy credentials from ``~/.netrc`` file if present.
 
+      Get HTTP Basic Auth credentials from ``~/.netrc`` file if present.
+
+      If ``NETRC`` environment variable is set, read from file specified
+      there rather than from ``~/.netrc``
```

Comment:
```suggestion
      Get HTTP Basic Auth credentials from :file:`~/.netrc` file if present.

      If :envvar:`NETRC` environment variable is set, read from file specified
      there rather than from :file:`~/.netrc`.
```

(proper markup + trailing period)

Reply:
The `:envvar:` role may generate a warning in Sphinx, though. In this case, you may need to either ignore it in `conf.py`, or declare it.